packages/@ourworldindata/grapher/src/timeline/TimelineComponent.tsx

```diff
@@ -278,6 +345,12 @@ export class TimelineComponent extends React.Component<TimelineComponentProps> {
                         ? controller.resetStartToMin()
                         : controller.resetEndToMax()
                 }
+                onMouseEnter={() => {
+                    if (this.shouldShowHoverTimeHandle) this.hoverTime = time
+                }}
```

Comment:
A slight change in behavior here is that the handle that comes up when hovering over the start time [here](http://staging-site-second-time-slider-handle/grapher/life-expectancy?tab=table&time=earliest) is now grey, instead of the blue it was before. That's because, in reality, it's two handles stacked on top of each other, and the `hoverHandle` is the top one.

We don't necessarily need to change that, it's just something I noticed.

<img width="199" height="86" alt="Image" src="https://github.com/user-attachments/assets/dbc3f865-41d7-45e3-a911-7c838638987b" />

Reply:
I think I'm ok with that!